baggysoapshop/templates/oscar/partials/footer.html

```diff
@@ -30,7 +30,10 @@ <h2 class="text-uppercase">Get In Touch</h2>
           </div>
           <div class="col-md-4 mb-5 mb-lg-0">
             <a href="{% url 'about' %}"><h2 class="text-uppercase title">About Us</h2></a>
-            <p class="lead mb-0">The Baggy Soap Company Limited is registered in England and Wales with company number 11090960.</p>
+            <p class="lead mb-0">The Baggy Soap Company Limited is registered in England and Wales with company number 11090960.
+              <br>
+              copyright &copy; <script> document.write(new Date().getFullYear()); </script>
```

Comment:
```suggestion
              <br /><br />
              Copyright &copy; <script> document.write(new Date().getFullYear()); </script>
```

Reply:
I think it looks better visually with a capital 'C' and another line break. Great to have it there though - thanks! :+1: